src/main/java/net/minecraftforge/server/command/CommandSetDimension.java

```diff
@@ -41,7 +42,7 @@
 public class CommandSetDimension
 {
     private static final SimpleCommandExceptionType NO_ENTITIES = new SimpleCommandExceptionType(new TranslationTextComponent("commands.forge.setdim.invalid.entity"));
-    private static final DynamicCommandExceptionType INVALID_DIMENSION = new DynamicCommandExceptionType(dim -> new TranslationTextComponent("commands.forge.setdim.invalid.dim", dim));
```

Comment:
Don't remove this, it is here to remember to reimplement it at some point. (Also with the requested change to the default implementation of `getPortalInfo` you should be able to revert all changes being made to this class.)